packages/amplify_api_plugin_interface/lib/src/types.dart

```diff
@@ -13,6 +13,7 @@
  * permissions and limitations under the License.
  */
 
+export 'GraphQL/GraphQLHelpers.dart';
```

Comment:
Does this file exist yet? If not, could this export be delayed (removed from this PR and re-added when the file exists) so all these exports resolve?

Reply:
Not yet, this was an artifact of cherry picking commits.